### PR TITLE
Signed-off-by: Scott Rosenberg scott@tera-sky.com

### PR DESCRIPTION
**Pull request by vrabbi**
## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: community.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ It would be great to hear from you especially if you have any of the above and w
 | [Hands-on Serverless with OpenFaaS and Python](https://gotochgo.com/2019/workshops/152) | Alex Ellis | Chicago, USA | 1-May-2019 |
 | [GOTO Chicago - Serverless Beyond the Hype](https://gotochgo.com/2019/sessions/728) | Alex Ellis | Chicago, USA | 30-Apr-2019 |
 | [Serverless Architecture Conf: OpenFaaS - Keeping Serverless Simple](https://serverless-architecture.io/serverless-platforms-technology) | John McCabe | The Hague, Netherlands | 9-Apr-2019 |
+| [openfaas in the vmware ecosystem - webinar](https://www.terasky.com/news/openfaas-in-the-vmware-ecosystem-webinar-april-3rd-2019/) | Scott Rosenberg | online from Israel | 3-Apr-2019 |
```
Comment (rgee0) [inline community.md]: Can we be sensitive to branding:

openfaas => OpenFaaS
vmware => VMware

**Comment by vrabbi** [comment]
How do I add this to the actual document. I added a line to the events table.

File: community.md (hunk 1 above)
Comment (vrabbi) [reply community.md]: updated in new commit

**Comment by alexellis** [comment]
Finally can we squash/reset the three commits and send them in signed-off?

**Comment by vrabbi** [comment]
@rgee0  can you help me with this. im not sure how to squash/reset the commits

**Comment by rgee0** [comment]
If you're doing it through the UI it might be easier to open a new PR and copy the content from this one.  Then close this one.

If you're doing it locally then on your branch try a `git rebase -i HEAD~3` and then mark the latest two commits for `fixup`.

**Pull request closed without merging**